src/styles/_base.css

```diff
@@ -64,41 +72,66 @@ header {
   font-size: large;
 }
 
-.header__search-bar {
-  border-radius: 5px;
-  border: 2px solid #cdfbbb;
-}
-
 .main-layout {
-  display: grid;
+  display: flex;
+  flex-wrap: wrap;
   grid-row: content;
+  grid-column: width;
   overflow-y: scroll;
-  background-color: bisque;
+  background-image: url(../icon/Background.png);
+  background-repeat: no-repeat;
+  background-size: contain;
 }
 .characters {
   display: flex;
   width: 100vw;
   justify-content: space-between;
+  align-items: center;
+  padding: 5px;
   background-color: var(--background-color-characters);
```

Comment:
Not sure you need to define the background-color here if you are changing it dynamically in the js

Reply:
Thank you for your comments! :)